### DISCUSSION TITLE
Wrong ESP32 Module, how to continue ?

### DISCUSSION THREAD

**Discussion by Dabbadoeber** [jomjol/AI-on-the-edge-device#1791] [General]
i also have ordered the ESP32 For this purpose after trail and error i found out, that i to have the wrong psram (only 2mb)
while ESP home says 4. So now i have 2 useless ESP32 devices on my desk, the cam is working with ESPhome and home assistant.
is it not possible to use the CAM of the ESP32 and move all the algorithms / AI part to a other device (docker, or something els) now this project is costing mee more then i wanted because i have to order ESP module until i have one that is working.

**Comment by caco3** [comment]
yes, this would be possible but is not supported by this project. You would have to do it yourself.

Bad/wrong hardware is a big issue, sadly.

Did you check https://github.com/jomjol/AI-on-the-edge-device/discussions/1732 ?

**Reply by Dabbadoeber** [reply]
Yes i checked it,, but you never know what u are going to get, even with ESP home flash the ESP32-CAM was not rock solid...
